src/components/ui/AppColorPicker.vue

```diff
@@ -327,6 +347,16 @@ export default class AppColorPicker extends Vue {
     this.relativeMove(newPosition)
     this.lastPointerPosition = newPosition
   }
+
+  convertValueRange (value: number, direction: 'in' | 'out') {
+    if (this.valueRange === 'absolute') return value
+
+    let factor = 1
+    if (this.valueRange === 'percentage') factor = 255
+    if (direction === 'out') factor = 1 / factor
+
+    return Math.round(value * factor * 1000) / 1000
+  }
```

Comment:
Not sure all this is really necessary, I've implemented it like this just in case there'll be more value ranges in the future, but I personally can't think of any (that would work with the number input, anyway). Using a boolean `usesPercentageValueRange` property instead would work fine too, and might save a tiny bit of memory